strawberry/types/type_resolver.py

```diff
@@ -296,64 +274,19 @@ def _get_fields(cls: Type) -> List[FieldDefinition]:
             field_definitions = {**field_definitions, **base_field_definitions}
 
     # then we can proceed with finding the fields for the current class
+    for field in dataclasses.fields(cls):
+        # TODO: Importing at top of file cause cyclical import problems. Maybe there's
+        #       a better place for this?
+        from strawberry.field import StrawberryField
```

Comment:
we can do this before the for?

Reply:
Oh oops. I meant to do that. I was trying to keep the import close to where it's used, but accidentally put it _too_ close.